File: WolfDen/wwwroot/ts/AdvanceWarsPlayer.ts
namespace TurnBasedBoardGame {
    export class AdvanceWarsPlayer{
        Pieces: Piece[];
        Id: string;
        Name: string;
        Color: string;
        Team: number;
        Defeated: boolean;
        _gold: number = 0;
        CO: CO;
        Listings: any;
        RepairCoordinator: RepairCoordinator;

        get Gold(): number {
            return this._gold;
        }

        set Gold(value: number) {
            this._gold = value;
        }
        
    }
}

File: WolfDen/Game/AdvanceWarsPlayer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using WolfDen.Game.Other;
using WolfDen.Game.Tiles;

namespace WolfDen.Game
{
    public class AdvanceWarsPlayer
    {
        public string Class { get { return GetType().Name; } }
        public string Id { get; set; }
        public List<Piece> Pieces = new List<Piece>();
        public string Name { get; set; }

        public string Color { get; set; } = "Orange";
        public byte Team { get; set; }
        public bool Defeated { get; set; }

        public int Gold { get; set; }
        public CO CO { get; set; }
        public string CoName { get; set; }
        public Dictionary<BuildingType, Dictionary<string, ProducingBuildingListing>> Listings => CO.Listings;
        public RepairCoordinator RepairCoordinator { get; set; }


        public AdvanceWarsPlayer()
        {
            RepairCoordinator = new RepairCoordinator(this);
        }
    }
}